src/main/webapp/WEB-INF/tags/instructor/feedbacks/softDeletedFeedbackSessionsTable.tag

```diff
@@ -25,8 +25,8 @@
   </thead>
   <c:forEach items="${softDeletedFsList.rows}" var="softDeletedSession" varStatus="i">
     <tr>
-      <td id="softdeletedcourseid${i.index}">${softDeletedSession.courseId}</td>
-      <td id="softdeletedsessionname${i.index}">${softDeletedSession.sessionName}</td>
+      <td id="softdeletedcourseid${i.index}" data-soft-deleted-course-id="${softDeletedSession.courseId}">${softDeletedSession.courseId}</td>
+      <td id="softdeletedsessionname${i.index}" data-soft-deleted-session-name="${softDeletedSession.sessionName}">${softDeletedSession.sessionName}</td>
```

Comment:
Previously when I mentioned the example I gave was not a good one because of the strong coupling here with too low-level details. 

Take for example you could instead do this to represent the data object (soft deleted session) as a whole:
```jsp
    <tr id="softdeletedsession${i.index}"
        data-course-id="${softDeletedSession.courseId}"
        data-session-name="${softDeletedSession.sessionName}">
```

Originally I was ok with you attaching the data to the "id" element because it is still somewhat reasonable to imply that all associated data can be attached to the unique "id" element. However, it seems like it can cause one to mistakenly think attaching data at such a low-level granularity is a good idea. The main issue is that we force a stronger coupling than we want, at a very low-level granularity.

To give an example, suppose we re-arrange the way data is presented in the table row. Then since the associated data are at such a low level of granularity, we have to move the data accordingly as well.

As you can see though, if we re-arrange how rows are presented the data will still be affected. However, this is the current compromise we are taking by using `data-*` attributes. As mentioned earlier, in the future we should use approaches such as AJAX to retrieve the data and populate them accordingly.